kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -173,11 +177,13 @@ export default {
       PointsIcon,
       AuthMessage,
       FilterTextbox,
+      CoreBanner,
       CoreSnackbar,
       CoreMenu,
       CoreMenuDivider,
       CoreMenuOption,
       CoreTable,
+      GlobalSnackbar,
```

Comment:
This thread is not a blocker for merge, but just have some thoughts to share about some code organization stuff.

I thinking this adds a smidgen of tech debt; however it is not easily avoided because of the need to have the CoreBanner specifically available to the User Auth plugin in any case. I thought about perhaps moving the `CoreBanner` component over to the User Auth plugin altogether, but I think that's more work than its worth.

I'm also hoping to avoid adding to the apiSpec as much as we can as it adds some bloat, especially for something not being used across multiple plugins.

To avoid that bloat perhaps we can make this an exception for our *no importing across packages* "rule". It feels slightly less problematic when we're reaching out for something in core than it would to import across plugins since `core` will always be there. Curious on thoughts from @marcellamaki and @rtibbles as well.

Finally my thought about moving `UserAuthLayout` into `core` is kind of a similar problem where rather than adding one-use components into the ApiSpec, we'd be adding a one-use component into the `core` app

Reply:
> I'm also hoping to avoid adding to the apiSpec as much as we can as it adds some bloat, especially for something not being used across multiple plugins.

This is what motivated me to suggest creating a package in `packages` for this kind of common functionality that is used in multiple plugins, but does not need to be exposed in the apiSpec as something that we are explicitly supporting for any arbitrary third party plugin.

See https://github.com/learningequality/kolibri/issues/5488

 and the additional follow up you created: https://github.com/learningequality/kolibri/issues/9589

Reply:
As a way to get us started on #5488  and #9589, a new package `kolibri-common` has been added under packages. This references shared components within the kolibri plugins. References added previously to the `apiSpec.js` have been reverted. This is minimalistic so comments to improve this are welcome. Thanks cc @nucleogenesis  @rtibbles